src/contexts/RainbowProvider/configs.tsx

```diff
@@ -1,69 +1,23 @@
-import { Chain, Wallet } from "@rainbow-me/rainbowkit"
-import "@rainbow-me/rainbowkit/styles.css"
-import {
-  braveWallet,
-  coinbaseWallet,
-  frameWallet,
-  imTokenWallet,
-  injectedWallet,
-  metaMaskWallet,
-  okxWallet,
-  rabbyWallet,
-  rainbowWallet,
-  safeWallet,
-  walletConnectWallet,
-} from "@rainbow-me/rainbowkit/wallets"
+import { getDefaultConfig } from "@rainbow-me/rainbowkit"
+import { injectedWallet, metaMaskWallet, okxWallet, rabbyWallet, safeWallet, walletConnectWallet, zerionWallet } from "@rainbow-me/rainbowkit/wallets"
+import { Chain, mainnet, scroll, scrollSepolia, sepolia } from "@wagmi/core/chains"
 import { parseUnits } from "ethers"
 import produce from "immer"
-import { configureChains, mainnet, sepolia } from "wagmi"
-import { jsonRpcProvider } from "wagmi/providers/jsonRpc"
-import { publicProvider } from "wagmi/providers/public"
 
-import { CHAIN_ID, ETH_SYMBOL, L2_NAME, RPC_URL } from "@/constants"
-import { networkType, requireEnv } from "@/utils"
+import { RPC_URL } from "@/constants"
+import { requireEnv } from "@/utils"
 
-interface WalletConfig {
-  name: string
-  wallet: Wallet
-  visible: boolean
-  fixedWallet?: boolean
-}
-
-const createWalletConfig = (name: string, walletFunction: () => Wallet, condition: boolean, fixedWallet?: boolean): WalletConfig => {
-  return {
-    name,
-    wallet: walletFunction(),
-    visible: condition,
-    fixedWallet,
-  }
-}
+const projectId = requireEnv("REACT_APP_CONNECT_WALLET_PROJECT_ID")
 
-export const scrollChain: Chain = {
-  id: CHAIN_ID.L2,
-  name: L2_NAME,
-  network: L2_NAME,
-  iconUrl: "https://scroll.io/logo.png",
-  iconBackground: "#fff",
-  nativeCurrency: {
-    decimals: 18,
-    name: L2_NAME,
-    symbol: ETH_SYMBOL,
-  },
-  rpcUrls: {
-    default: {
-      http: [RPC_URL.L2],
-    },
-    public: {
-      http: [RPC_URL.L2],
-    },
-  },
-  blockExplorers: {
-    default: { name: "Scrollscan", url: requireEnv("REACT_APP_EXTERNAL_EXPLORER_URI_L2") },
+const wallets = [
+  {
+    groupName: "Popular",
+    wallets: [walletConnectWallet, metaMaskWallet, okxWallet, rabbyWallet, safeWallet, zerionWallet, injectedWallet],
```

Comment:
Popular wallets should be specific, and I think injectedWallet should not be here.
![CleanShot 2024-04-03 at 09 10 00@2x](https://github.com/scroll-tech/frontends/assets/21138718/5bd8d9d9-96cd-421e-b781-417028ffc4f2)